enumerable.rb

```diff
@@ -1,20 +1,4 @@
-module MyEnumerable
-  def all?
-    each { |element| return false unless yield(element) }
-    true
-  end
-
-  def any?
-    each { |element| return true if yield(element) }
-    false
-  end
-
-  def filter
-    result = []
-    each { |element| result << element if yield(element) }
-    result
-  end
-end
+require_relative 'MyEnumerable'
```

Comment:
- [ ] Great job following the changes suggested by the previous reviewer! 🚀 However, please make sure that your code is working before submitting it again! The `require_relative` method expects the name of the file, in this case, `module` instead of the actual name of the module `MyEnumerable`. Please consider fixing this! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/234884032-e75b44a4-3423-47a8-8971-6c070c38cf07.png)